xpti/include/xpti_trace_framework.hpp

```diff
@@ -281,18 +240,18 @@ class PlatformHelper {
 
   bool checkTraceEnv() {
     bool enabled = false;
-    std::wstring env = getEnvironmentVariable("XPTI_TRACE_ENABLE");
+    std::string env = getEnvironmentVariable("XPTI_TRACE_ENABLE");
     if (env.empty()) {
       enabled = true;
       return enabled;
     }
-    if (env == L"true") {
+    if (env == "true") {
       enabled = true;
-    } else if (env == L"false") {
+    } else if (env == "false") {
       enabled = false;
-    } else if (env == std::to_wstring(1)) {
+    } else if (env == std::to_string(1)) {
```

Comment:
Do you mean
```suggestion
    } else if (env == "1") {
```
? :-)
Actually simplify further so it is more understandable:
```
if (env == "true" || env == "1")
  return true;
```

Reply:
Done!